### PR TITLE
feat: update system status

### LINKED ISSUE #1404: We can get the backend version and git hash at the front end.
# Feature request

## Please describe your feature

Hello, there is no way to get the backend version and the git hash at the front end.
But I think it's in need.

## Describe the solution you'd like

Backend adds an API like `/apisix/admin/version" to return the version and git hash.

### PR DESCRIPTION
**Pull request by LiteSun**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [ ] New feature provided
- [x] Improve performance
- [ ] Backport patches

- Related issues
resolves #1404 
___
image

### CODE REVIEW

File: web/src/pages/ServerInfo/List.tsx [hunk 1]
```diff
@@ -51,58 +53,84 @@ const ServerInfo: React.FC = () => {
...
+        <div className={styles.wrap} >
+          {nodeList.length > 0 && (
+            <table className={styles.table}>
+              <tbody>
+                <tr>
+                  <td>commit_hash</td>
+                  <td>{commitHash}</td>
+                </tr>
+                <tr>
+                  <td>dashboard_version</td>
```
Comment (juzhiyuan) [inline web/src/pages/ServerInfo/List.tsx:75]:
```suggestion
                  <td>Version</td>
```

File: web/src/pages/ServerInfo/List.tsx [hunk 2]
```diff
@@ -51,58 +53,84 @@ const ServerInfo: React.FC = () => {
...
-                </Option>
-              ))}
+    <PageContainer title={formatMessage({ id: 'page.systemStatus.pageContainer.title' })}>
+      <Card title={formatMessage({ id: 'page.systemStatus.dashboardInfo' })} bodyStyle={{ padding: 0 }} style={{ marginBottom: 15 }}>
+        <div className={styles.wrap} >
+          {nodeList.length > 0 && (
+            <table className={styles.table}>
+              <tbody>
+                <tr>
+                  <td>commit_hash</td>
```
Comment (juzhiyuan) [inline web/src/pages/ServerInfo/List.tsx:71]:
```suggestion
                  <td>Commit Hash</td>
```

File: web/src/pages/ServerInfo/locales/en-US.ts [hunk 3]
```diff
@@ -15,8 +15,10 @@
  * limitations under the License.
  */
 export default {
-  'page.serverinfo.pageContainer.title': 'Server Info',
-  'page.serverinfo.select.placeholder': 'Please select node',
-  'page.serverinfo.desc': 'The relevant plugin need to be enabled to report server info.',
-  'page.serverinfo.link': 'How to enable?',
+  'page.systemStatus.pageContainer.title': 'System Status',
+  'page.systemStatus.select.placeholder': 'Please select node',
```
Comment (juzhiyuan) [inline web/src/pages/ServerInfo/locales/en-US.ts:19]:
```suggestion
  'page.systemStatus.select.placeholder': 'Please select a apache apisix node',
```

File: web/src/pages/ServerInfo/locales/en-US.ts [hunk 4]
```diff
@@ -15,8 +15,10 @@
...
+  'page.systemStatus.desc': 'The relevant plugin need to be enabled to report server info.',
```
Comment (juzhiyuan) [inline web/src/pages/ServerInfo/locales/en-US.ts:20]:
```suggestion
  'page.systemStatus.desc': 'The relevant plugin needs to be enabled to report server info.',
```

File: web/src/pages/ServerInfo/typing.d.ts [hunk 5]
```diff
@@ -24,4 +24,9 @@ declare namespace ServerInfoModule {
     hostname: string;
     version: string;
   };
+  
+  type DashboardInfo = {
```
Comment (juzhiyuan) [inline web/src/pages/ServerInfo/typing.d.ts:28]:
```ts
type T = Record<string, string>
```

File: web/src/pages/ServerInfo/List.tsx [hunk 6]
```diff
@@ -51,58 +53,84 @@ const ServerInfo: React.FC = () => {
         setData(list[0]);
       }
     });
+
+    fetchVersion().then(({ commit_hash, version }) => {
```
Comment (juzhiyuan) [inline web/src/pages/ServerInfo/List.tsx:57]: We could use a loop

**Review by juzhiyuan — APPROVED** [review]
LGTM

**Comment by membphis** [comment]
ci failed, @nic-chen do you know the reason?

**Comment by nic-chen** [comment]
> ci failed, @nic-chen do you know the reason?

it has been fixed. @LiteSun  please merge master to your branch, thanks.

File: web/src/pages/ServerInfo/List.tsx [hunk 7]
```diff
@@ -51,58 +53,84 @@ const ServerInfo: React.FC = () => {
...
-            >
-              {nodeList.map((item) => (
-                <Option key={item.hostname} value={item.id}>
-                  {item.hostname}
-                </Option>
-              ))}
+    <PageContainer title={formatMessage({ id: 'page.systemStatus.pageContainer.title' })}>
+      <Card title={formatMessage({ id: 'page.systemStatus.dashboardInfo' })} bodyStyle={{ padding: 0 }} style={{ marginBottom: 15 }}>
+        <div className={styles.wrap} >
+          {nodeList.length > 0 && (
```
Comment (liuxiran) [inline web/src/pages/ServerInfo/List.tsx:67]: just to make sure that wether the dashboard `git hash` `version`  and `APISIX info` are Independent, which means: users will see the dashboard `git hash` and `version` no matter  whether `server-info` plugin is enabled or not

if so , IMHO, we should remote `nodeList.lenght > 0`

Comment (juzhiyuan) [reply web/src/pages/ServerInfo/List.tsx:67]: oh yes, ping @LiteSun to modify.

Comment (juzhiyuan) [reply web/src/pages/ServerInfo/List.tsx]: just removed, cc @liuxiran

**Review by liuxiran — APPROVED** [review]

**Pull request merged**